.github/workflows/python-app.yml

```diff
@@ -28,11 +28,10 @@ jobs:
       postgres:
         image: postgres:12.2
         env:
-          POSTGRES_USER: tcf_django
-          POSTGRES_PASSWORD: s3kr1t
-          POSTGRES_DB: tcf_db
-        ports:
-          - 5432:5432
+          POSTGRES_USER: ${{ secrets.DB_USER }}
+          POSTGRES_PASSWORD: ${{ secrets.DB_PASSWORD }}
+          POSTGRES_DB: ${{ secrets.DB_NAME }}
+          POSTGRES_PORT: ${{ secrets.DB_PORT }}

```

Comment:
Github won't let me check what the values of the repo secrets are; are these parameters for connecting to prod DB or establishing a local test DB? Is it advisable to be running tests against the prod DB?

Reply:
These are the env vars in `.env.ci`, which you can find in the Google Drige folder I mentioned in the description.